tests/framework/test_Importance_Sampler.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" ?>
```

Comment:
I think this test would be better placed in `raven/tests/framework/Samplers/`

Reply:
Just wanted to double-check on this. I placed this test where the other sampler tests are located.